specs/commandline/version.md

```diff
@@ -37,7 +37,7 @@ notation version
 An example output:
 
 ```text
-Notation: Notary v2, A tool to sign, store, and verify artifacts.
+Notation: A tool to sign, store, and verify artifacts.
```

Comment:
Suggest removing "store" from the sentence and use the same unified sentence as `Notation - a tool to sign and verify artifacts. `